LoadEnvVariables/LoadEnvVariables.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DotNetEnv;
+using Newtonsoft.Json.Linq;
+
+namespace LoadEnvVariables;
+
+public class AzureEnvManager
```

Comment:
```suggestion
namespace AzdLibrary;

public class AzdEnvironment
```

Also change the project and folder names